toolchain/autoupdate_testdata.py

```diff
@@ -65,6 +67,10 @@ def main() -> None:
         "--",
         "--autoupdate",
     ]
+    if args.print_slowest_tests:
+        argv += ["--print_slowest_tests", args.print_slowest_tests]
```

Comment:
I think we should turn this off by default for `autoupdate_testdata.py`. We've worked to keep the output produced by that script minimal when updates failed, and when that script is used as part of a regular development workflow I don't think this is going to be interesting information most of the time.

Reply:
Can you elaborate what you mean by "updates failed"? If you mean "crashed", note this isn't printed on crash. I'm not sure what else you'd call a "fail" state.

Reply:
And note, I think the routine interesting part is to nudge people to write faster tests.

Reply:
Err, sorry, I meant "when updates succeeded"! That is, we just produce a line of `.`s and `!`s in that case (plus some noise from bazel we can't turn off any more).

Reply:
I'm taking this as continued nudging for less output. Trimmed.